CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ We’ve also made fixes in the following pull requests:
 
 - [#2549: Fix header with product name focus and hover state length](https://github.com/alphagov/govuk-frontend/pull/2549)
 - [#2573: Better handle cases where `$govuk-text-colour` is set to a non-colour value](https://github.com/alphagov/govuk-frontend/pull/2573)
-- [#2590: Maxlength attribute should not be present on textarea after character-count JavaScript has kicked in](https://github.com/alphagov/govuk-frontend/pull/2590)
+- [#2590: `maxlength` attribute should not be present on `textarea` after character count JavaScript has kicked in](https://github.com/alphagov/govuk-frontend/pull/2590)
```

Comment:
Wonder could we reword this title so that it begins with a verb ('Remove `maxlength` attribute...' or something like that)?

Also, could we say "started to work" (or similar) instead of "kicked in"?

Reply:
Sound like sensible changes to me 👍

Reply:
@querkmachine Thanks :)